documentation/manual/src/main/asciidoc/modules/infinispan.asciidoc

```diff
@@ -512,6 +515,14 @@ Start with the default configuration and see if that fits you.
 You can fine tune these setting when you are closer to going in production.
 ====
 
+====
+
+In order to enable `Clustered Counter` support we have defined a `counters` tag.
+Cluster counter support it is used to implement a reliable and consist stategy to generate unique source ID value.
```

Comment:
Just a few typos noted:
s/it//
s/consist/consistent/
s/value/values/ ?

Reply:
s/tranport/transport/

Reply:
> Cluster counter support it is used to implement

I think some quotes are missing? Currently I can't understand this.